src/prop/sat_solver.h

```diff
@@ -170,8 +170,23 @@ class CDCLTSatSolver : public SatSolver
    */
   virtual std::vector<Node> getOrderHeap() const = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is PROOF.
+   * Returns a complete proof computed by this SAT solver.
+   */
   virtual std::shared_ptr<ProofNode> getProof() = 0;
 
+  /**
+   * Get proof, which is used if prop-proof-mode is SKETCH.
+   * Returns a rule r and additional arguments args such that the final proof
```

Comment:
Use `@return` and describe parameters with `@param` (generally, for all changes in this PR). We should make this a habit as part of the effort of improving and making the internal documentation consistent.